server/container_create.go

```diff
@@ -286,9 +295,14 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 
 	logrus.Debugf("pod container state %+v", podInfraState)
 
-	ipcNsPath := fmt.Sprintf("/proc/%d/ns/ipc", podInfraState.Pid)
-	if err := specgen.AddOrReplaceLinuxNamespace("ipc", ipcNsPath); err != nil {
-		return nil, err
+	for nsType, nsFile := range map[rspec.NamespaceType]string{
+		rspec.PIDNamespace: "pid",
```

Comment:
We can take this out for now as k8s is still discussing viability and it may be pushed to next release.